src/csproj.js

```diff
@@ -114,7 +134,7 @@ const parseProjectSync = (filePath, options) => {
     const packagesLocation = path.join(projDir, 'packages.config');
 
     const packages = helpers.fileExistsSync(packagesLocation) && parsePackagesSync(packagesLocation);
-    result.packages = packages || [];
+    result.packages = mergePackages(result.packages, packages);
```

Comment:
Similar suggestion for here. Can we please only parse packages.config if we can't read any packages from the csproj file?